applications/connecting_applications_to_services/getting-started-with-service-binding.adoc

```diff
@@ -0,0 +1,37 @@
+[id="getting-started-with-service-binding"]
+= Getting started with service binding
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: getting-started-with-service-binding
+
+toc::[]
+
+{servicebinding-title} manages the data plane for workloads and backing services. This guide provides instructions with examples to help you create a database instance, deploy an application and use {servicebinding-title} to create a binding connection between the application and the database service.
+
+// Prerequisites for getting started with Service Binding Operator
+[discrete]
+== Prerequisites
+
+* You have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have installed `oc` CLI.
+* You have installed PostgreSQL `psql` CLI.
+* You have installed {servicebinding-title} from OperatorHub.
+* You have installed the Crunchy Postgres for Kubernetes Operator from OperatorHub using the *v5* Update channel. The installed Operator is available in an appropriate namespace, for example, `my-postgresql` namespace.
+
+//Creating a PostgreSQL database instance
+include::modules/sbo-creating-a-postgresql-database-instance.adoc[leveloffset=+1]
+
+//Creating appropriate `ClusterRole` and ClusterRoleBinding` resources
+include::modules/sbo-creating-appropriate-clusterrole-and-clusterrolebinding-resources.adoc[leveloffset=+1]
```

Comment:
This section is not necessary since crunchy postgres is supported by default via #1043. @pedjak please confirm